### ISSUE TITLE
CT cannot read property width of null

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gas-properties#175]
Happens in Gasses Intro too
```
gas-properties : fuzz : built : load
Query: fuzz&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1241677
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:11334)
    at Function.value (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:38026)
    at t [as constructor] (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1241525)
    at t.i (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1261396)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1310209)
    at t.s (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1343742)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1366999)
    at t.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1369641)
    at t.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:840275)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : fuzz : built : run
Query: fuzz&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1241677
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:11334)
    at Function.value (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:38026)
    at t [as constructor] (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1241525)
    at t.i (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1261396)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1310209)
    at t.s (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1343742)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1366999)
    at t.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:1369641)
    at t.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/build/phet/gas-properties_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:923:840275)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : fuzz : built-phet-io : load
Query: fuzz&memoryLimit=1000&phetioStandalone
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1247429
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:11334)
    at Function.value (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:43982)
    at t [as constructor] (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1247277)
    at t.i (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1267148)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1315961)
    at t.s (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1349494)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1372751)
    at t.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1375393)
    at t.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:846027)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : fuzz : built-phet-io : run
Query: fuzz&memoryLimit=1000&phetioStandalone
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1247429
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:11334)
    at Function.value (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:43982)
    at t [as constructor] (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1247277)
    at t.i (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1267148)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1315961)
    at t.s (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1349494)
    at new t (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1372751)
    at t.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:1375393)
    at t.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335-phet-io/gas-properties/build/phet-io/gas-properties_all_phet-io.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000&phetioStandalone:921:846027)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : fuzz : require.js : load
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570545933152:263:50
    at new Multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Multilink.js?bust=1570545933152:50:16)
    at Function.multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Property.js?bust=1570545933152:465:14)
    at new Dialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570545933152:250:43)
    at new OopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/scenery-phet/js/OopsDialog.js?bust=1570545933152:64:7)
    at new GasPropertiesOopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/GasPropertiesOopsDialog.js?bust=1570545933152:39:7)
    at new IdealGasLawScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/IdealGasLawScreenView.js?bust=1570545933152:310:44)
    at new IdealScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/view/IdealScreenView.js?bust=1570545933152:51:7)
    at IdealScreen.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/IdealScreen.js?bust=1570545933152:40:35)
    at IdealScreen.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/joist/js/Screen.js?bust=1570545933152:260:25)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : fuzz : require.js : run
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570545933152:263:50
    at new Multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Multilink.js?bust=1570545933152:50:16)
    at Function.multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Property.js?bust=1570545933152:465:14)
    at new Dialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570545933152:250:43)
    at new OopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/scenery-phet/js/OopsDialog.js?bust=1570545933152:64:7)
    at new GasPropertiesOopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/GasPropertiesOopsDialog.js?bust=1570545933152:39:7)
    at new IdealGasLawScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/IdealGasLawScreenView.js?bust=1570545933152:310:44)
    at new IdealScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/view/IdealScreenView.js?bust=1570545933152:51:7)
    at IdealScreen.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/IdealScreen.js?bust=1570545933152:40:35)
    at IdealScreen.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/joist/js/Screen.js?bust=1570545933152:260:25)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : phet-io-tests : assert
7 out of 9 tests passed. 2 failed.
SimTests: gas-properties: iframe api failed:
Uncaught Error: Uncaught TypeError: Cannot read property 'width' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/phet-io-wrappers/js/tests/SimTests.js?bust=1570545016111:341

SimTests: gas-properties: event indexing failed:
Uncaught Error: Uncaught TypeError: Cannot read property 'width' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/phet-io-wrappers/js/tests/SimTests.js?bust=1570545016111:531


id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : phet-io-tests : no-assert
7 out of 9 tests passed. 2 failed.
SimTests: gas-properties: iframe api failed:
Uncaught Error: Uncaught TypeError: Cannot read property 'width' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/phet-io-wrappers/js/tests/SimTests.js?bust=1570543375907:341

SimTests: gas-properties: event indexing failed:
Uncaught Error: Uncaught TypeError: Cannot read property 'width' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/phet-io-wrappers/js/tests/SimTests.js?bust=1570543375907:531


id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : xss-fuzz : load
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570546379533:263:50
    at new Multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Multilink.js?bust=1570546379533:50:16)
    at Function.multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Property.js?bust=1570546379533:465:14)
    at new Dialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570546379533:250:43)
    at new OopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/scenery-phet/js/OopsDialog.js?bust=1570546379533:64:7)
    at new GasPropertiesOopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/GasPropertiesOopsDialog.js?bust=1570546379533:39:7)
    at new IdealGasLawScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/IdealGasLawScreenView.js?bust=1570546379533:310:44)
    at new IdealScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/view/IdealScreenView.js?bust=1570546379533:51:7)
    at IdealScreen.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/IdealScreen.js?bust=1570546379533:40:35)
    at IdealScreen.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/joist/js/Screen.js?bust=1570546379533:260:25)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
gas-properties : xss-fuzz : run
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570546379533:263:50
    at new Multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Multilink.js?bust=1570546379533:50:16)
    at Function.multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Property.js?bust=1570546379533:465:14)
    at new Dialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570546379533:250:43)
    at new OopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/scenery-phet/js/OopsDialog.js?bust=1570546379533:64:7)
    at new GasPropertiesOopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/GasPropertiesOopsDialog.js?bust=1570546379533:39:7)
    at new IdealGasLawScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/IdealGasLawScreenView.js?bust=1570546379533:310:44)
    at new IdealScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/view/IdealScreenView.js?bust=1570546379533:51:7)
    at IdealScreen.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/IdealScreen.js?bust=1570546379533:40:35)
    at IdealScreen.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/joist/js/Screen.js?bust=1570546379533:260:25)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
```

**Comment by pixelzoom** [comment]
The relevant stack trace is:

```
gas-properties : xss-fuzz : run
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught TypeError: Cannot read property 'width' of null
TypeError: Cannot read property 'width' of null
    at https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570546379533:263:50
    at new Multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Multilink.js?bust=1570546379533:50:16)
    at Function.multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/axon/js/Property.js?bust=1570546379533:465:14)
    at new Dialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/sun/js/Dialog.js?bust=1570546379533:250:43)
    at new OopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/scenery-phet/js/OopsDialog.js?bust=1570546379533:64:7)
    at new GasPropertiesOopsDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/GasPropertiesOopsDialog.js?bust=1570546379533:39:7)
    at new IdealGasLawScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/common/view/IdealGasLawScreenView.js?bust=1570546379533:310:44)
    at new IdealScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/view/IdealScreenView.js?bust=1570546379533:51:7)
    at IdealScreen.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/gas-properties/js/ideal/IdealScreen.js?bust=1570546379533:40:35)
    at IdealScreen.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1570538357335/joist/js/Screen.js?bust=1570546379533:260:25)
id: Bayes Chrome
Approximately 10/8/2019, 6:39:17 AM
```

@zepumph has been working in Dialog, e.g. https://github.com/phetsims/sun/issues/525.  I suspect that this is related.

**Comment by zepumph** [comment]
Yes sorry I didn't see this until now. I committed a fix and explained it over in https://github.com/phetsims/sun/issues/525#issuecomment-539613896. Please pull sun and confirm fixed @pixelzoom. The changes are out for your review over in that issue. Feel free to close this if you see it is fixed locally.

**Comment by pixelzoom** [comment]
Fix over in phetsims/sun#525  looks reasonable.  I'll close this when CT is green.

**Comment by pixelzoom** [comment]
CT is green. Closing.
